### ISSUE TITLE
Overseerr

### ISSUE THREAD

**Issue by t-abraham** [TrimVis/butlarr#5]
Hi I see you have connected the direct APIs of the arr apps but wont it be easier and better to connect it with overseerr in that case one can set up multiple arr apps in overseerr without hampering the script or yours. Just a thought

**Comment by TrimVis** [comment]
I have never worked with Overseer, but from what I can tell, it only integrates with Sonarr and Radarr (and Plex), right?

If it's only these two plugins, I don't see the immediate benefit of switching to overseer instead, as both of these are already integrated quite well :)

**Comment by TrimVis** [comment]
What do you mean by "without hampering the script or yours"?
As long as Overseer also lets the underlying Arr apps know when something is monitored, these two shouldn't really interfere with one another.
This bot is basically another way to interact with the Sonarr and Radarr interface.

**Comment by ghostkernel-core** [comment]
> I have never worked with Overseer, but from what I can tell, it only integrates with Sonarr and Radarr (and Plex), right?
> 
> If it's only these two plugins, I don't see the immediate benefit of switching to overseer instead, as both of these are already integrated quite well :)

yes you are right... I was targeting to the users who use overseerr with Sonarr and Radarr (and Plex). If the bot interracts with overseerr directly then no need to add Sonarr and Radarr apis... just another approach.

**Comment by ghostkernel-core** [comment]
> What do you mean by "without hampering the script or yours"? As long as Overseer also lets the underlying Arr apps know when something is monitored, these two shouldn't really interfere with one another. This bot is basically another way to interact with the Sonarr and Radarr interface.

what i meant is by "without hampering the script or yours"? is that instead of expossing the arr apis to multiple handlers, its better to expose to one and use that ones api to the bot. just a thought...

**Comment by TrimVis** [comment]
I will leave this open for now and think about it some more.

I saw that Overseer has some of its own abstractions (like user management, etc.), which I haven't had to worry about so far.
So I will keep this on the back burner for now, but in general, it definitely would be a usable plugin to add :)

**Comment by freakshock88** [comment]
A possible advantage of integrating with overseerr is that the server admin can have more control about what media ends up in the server, since there are things supported like admin approvals, request limits per user etc.

**Comment by maximecharriere** [comment]
With Overseerr you can manage people request. The Telegram Bot would just add a request to Overseerr that the admin can accept/deny.  
Take a look at https://github.com/thomst08/requestrr feature.

Edit:
Exist already here : https://github.com/LetsGoDude/OverseerrRequestViaTelegramBot
Maybe it's an idea to merge both project?
